src/static/devika/index.html

```diff
@@ -1,202 +1,15 @@
-<!DOCTYPE HTML>
+<!DOCTYPE html>
 <html lang="en">
-
 <head>
-    <meta charset="UTF-8">
-    <title>Lab 0 - Activity 2</title>
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="./style.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Stacked Bar Chart</title>
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://d3js.org/d3.v7.min.js"></script>
 </head>
-
 <body>
-    <div id="main">
```

Comment:
Can you embed HTML element changes to js file as well like css?
Since the entire webpage is using bootstrap, we were using id to get the graph to the entire web design.
```

          <div class="col-md-7 d-flex justify-content-center align-items-start">
            <div id="YOUR_ID_HERE" class="w-200 h-auto border rounded shadow bg-white" style="min-height: 300px;"></div>
          </div>
```